#15163 breaks windows compile

### Describe the bug

Can't compile darktable on windows since #15163 was merged.

### Steps to reproduce

1. Download current master on windows
2. Try and compile
3. Observe the error in src/gui/about.c saying that "tools/darktable_authors.h" couldn't be found.

### Expected behavior

darktable should compile without error

### Logfile | Screenshot | Screencast

_No response_

### Commit

_No response_

### Where did you install darktable from?

darktable.org

### darktable version

Current master

### What OS are you using?

Windows

### What is the version of your OS?

Win 10

### Describe your system?

Windows in a VM

### Are you using OpenCL GPU in darktable?

None

### If yes, what is the GPU card and driver?

N/A

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

_No response_

I always do a clean build.  Been bitten too many times.

Here's the error output

```
[ 33%] Building C object bin/CMakeFiles/lib_darktable.dir/dtgtk/thumbnail_btn.c.obj
[ 33%] Building C object bin/CMakeFiles/lib_darktable.dir/dtgtk/thumbtable.c.obj
[ 33%] Building C object bin/CMakeFiles/lib_darktable.dir/dtgtk/togglebutton.c.obj
[ 33%] Building C object bin/CMakeFiles/lib_darktable.dir/gui/about.c.obj
C:/msys64/home/bill/src/darktable/src/gui/about.c: In function 'darktable_show_about_dialog':
C:/msys64/home/bill/src/darktable/src/gui/about.c:52:10: fatal error: tools/darktable_authors.h: No such file or directory
   52 | #include "tools/darktable_authors.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [bin/CMakeFiles/lib_darktable.dir/build.make:1762: bin/CMakeFiles/lib_darktable.dir/gui/about.c.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:2287: bin/CMakeFiles/lib_darktable.dir/all] Error 2
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1046,6 +1046,9 @@ if(USE_LIBRAW AND NOT (DONT_USE_INTERNAL_LIBRAW AND libraw_FOUND))
   target_link_libraries(lib_darktable PRIVATE libraw::libraw)
 endif()
 
+#
+# generate the authors file
+#
 add_custom_command(
   DEPENDS ${CMAKE_SOURCE_DIR}/tools/authors_h.sh ${CMAKE_SOURCE_DIR}/AUTHORS
   OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/tools/darktable_authors.h

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -932,6 +932,7 @@ set_source_files_properties(${DARKTABLE_BINDIR}/version_gen.c PROPERTIES GENERAT
 add_dependencies(lib_darktable generate_conf)
 add_dependencies(lib_darktable generate_version)
 add_dependencies(lib_darktable generate_preferences)
+add_dependencies(lib_darktable generate_authors_h)
 
 if(APPLE)
   set_target_properties(lib_darktable PROPERTIES MACOSX_RPATH TRUE)

--- a/src/libs/CMakeLists.txt
+++ b/src/libs/CMakeLists.txt
@@ -56,7 +56,6 @@ add_library(ioporder MODULE "ioporder.c")
 add_library(viewswitcher MODULE "tools/viewswitcher.c")
 
 add_library(darktable_label MODULE "tools/darktable.c")
-add_dependencies(darktable_label generate_authors_h)
 
 add_library(colorlabels MODULE "tools/colorlabels.c")
 add_library(ratings MODULE "tools/ratings.c")

